Tasks/Common/webdeployment-common/utility.ts

```diff
@@ -183,6 +183,12 @@ export async function generateTemporaryFolderForDeployment(isFolderBasedDeployme
     if(isFolderBasedDeployment || packageType === PackageType.jar) {
         tl.debug('Copying Web Packge: ' + webDeployPkg + ' to temporary location: ' + folderPath);
         copyDirectory(webDeployPkg, folderPath);
+        if(packageType === PackageType.jar) {
+            let src = path.join(folderPath, getFileNameFromPath(webDeployPkg));
+            let dest = path.join(folderPath, "app.jar")
+            tl.debug("Renaming " + src + " to " + dest);
+            fs.rename(src, dest);
```

Comment:
Shouldn't we use fs.[renameSync ](https://nodejs.org/api/fs.html#fs_fs_renamesync_oldpath_newpath)?

Reply:
done